web/packages/teleterm/src/ui/ConnectMyComputer/DocumentConnectMyComputer/Status.tsx

```diff
@@ -163,19 +163,23 @@ export function Status(props: { closeDocument?: () => void }) {
           css={`
             display: block;
           `}
+          details={
+            <>
+              {isAgentConfiguredAttempt.statusText} <br />
+              You can try to{' '}
+              <Link
+                onClick={replaceWithSetup}
+                css={`
+                  cursor: pointer;
+                `}
+              >
+                run the setup
+              </Link>{' '}
```

Comment:
Consider also moving the link into the action button, like I did with the cluster alerts: https://github.com/gravitational/teleport/pull/45973/files#diff-d240fedc3624648199203972480e35b252b3842b1573aac9f20c50dc76a8995d

Reply:
Good idea. 
However, I was not able to apply it everywhere The Connect My Computer screen is quite narrow, and the button took to much place:
<img width="633" alt="image" src="https://github.com/user-attachments/assets/8c31ff16-88fb-4e8d-b4cc-3460de8922e9">
vs
<img width="633" alt="image" src="https://github.com/user-attachments/assets/faf542f0-d8a9-4ddd-97e7-cfe700748e4e">

But I changed it where it made sense.